src/components/popovers/menu.stories.mdx

```diff
@@ -0,0 +1,136 @@
+import { Meta, Preview, Story, Props } from "@storybook/addon-docs/blocks";
+import {
+  withKnobs,
+  array,
+  text,
+  boolean,
+  optionsKnob as options
+} from "@storybook/addon-knobs";
+import { action } from '@storybook/addon-actions';
+
+import Menu from './Menu';
+import MenuItem from './MenuItem';
+
+import Button from '../interactive/Button';
+
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+
+import CloneIcon from 'lib/icons/react/CloneIcon';
+import TrashIcon from 'lib/icons/react/TrashIcon';
+import EditIcon from 'lib/icons/react/EditIcon';
+import FundingIcon from 'lib/icons/react/FundingIcon';
+
+<Meta title="Popovers|Menu" component={Menu} decorators={[withKnobs]} />
+
+# Menu
+
+Renders a basic keyboard navigable dropdown menu following the [WAI-ARIA pattern for "menubutton"](https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton).
+
+### When to use Menu
+Only use a `Menu` for dropdowns that contain actions the user can take. If a dropdown includes any other type of content (forms, arbitrary content), use a `Popover` component instead.
+
+### Positioning
+The menu will automatically position above or below the trigger element based on its position in the viewport.
+
+<Story name="Knobs" parameters={{ docs: { disable: true }}}>
+  <div style={{ padding: '2rem' }}>
+    <Menu
+      trigger={<Button label={text('label', 'Menu trigger')} hasCaret />}>
+      {array('Menu Items', ['First', 'Second', 'Third', 'Fourth']).map((item) => (
+        <MenuItem onSelect={ action(`clicked ${item}`)}>{item}</MenuItem>
+      ))}
+    </Menu>
+  </div>
+</Story>
+
+<Preview isExpanded>
+  <Menu trigger={<Button label="Menu trigger" hasCaret />}>
+    <MenuItem onSelect={() => { alert('copy') }}>
+      Copy
+    </MenuItem>
+    <MenuItem onSelect={() => { alert('delete') }}>
+      Delete
+    </MenuItem>
+  </Menu>
+</Preview>
+
+## Menu Props
+
+<Props of={Menu} />
+
+## MenuItem Props
+
+<Props of={MenuItem} />
+
+## Trigger element
+
+The Menu trigger can be any JSX, but a `Button` with the `hasCaret` prop enabled is recommended.
```

Comment:
Do we want to recommend the DropdownButton be used here instead (as well as use that component as the example component throughout this story)?

Reply:
Over the last few weeks, my thinking on how we classify these popovers has shifted. Might be worth a deeper discussion. Basically, when in doubt, I think we should draw the lines using WAI-ARIA guidelines.

### Menubutton pattern
A menubutton is a button that triggers a menu. The menu is a list of actions to be taken. There is no "current selection", because this pattern is not about selecting a value from a list of options.

### Select pattern
A select pattern is a button or input that triggers a list of items that can be selected (they are not actions, just values to be selected). The button or input that triggers the selection list should display some information about the current selection.

The `DropdownButton` accepts arbitrary JSX because it must support showing something about a current selection. It also resembles an input more than a button, which is appropriate for a select-like pattern. It should be baked into some kind of `CustomSelect` component that mimics native `select` interaction. We may want to consider making `DropdownButton` a private component that only a `CustomSelect` would use.

The `Menu` component is following the menubutton pattern, which means the button that triggers the menu does not change based on actions taken within the menu. There is no "current selection" to show. That's why I'm recommending a plain `Button` here.

Reply:
Makes sense 👍 I'm going to add something to components weekly about this as this may have unforeseen design ramifications.